uni/lib/utils/navbar_items.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+
+enum NavbarItem {
+  navPersonalArea(Icons.home_outlined, 'area'),
+  navAcademicPath(
+    Icons.school_outlined,
+    'percurso_academico',
+  ),
+  navRestaurants(
+    Icons.local_cafe_outlined,
+    'restaurantes',
+  ),
+  navFaculty(
+    Icons.domain_outlined,
+    'faculdade',
+  ),
+  navTransports(
+    Icons.map_outlined,
+    'transportes',
+  );
+
+  const NavbarItem(this.icon, this.route);
+
+  final IconData icon;
+  final String route;
+
+  BottomNavigationBarItem toBottomNavigationBarItem() {
+    return BottomNavigationBarItem(
+      icon: Icon(icon),
+      label: '',
+    );
+  }
+}
```

Comment:
Can you use the NavigationItem enum to make it more consistent across the codebase?

Reply:
Yes, but not with `route`, as the value of the `route` field in the `NavigationItem` enum is not available at compile time. But storing the `NavigationItem` instance is possible.